### DISCUSSION TITLE
Can we have tooltip as bottom left ?

### DISCUSSION THREAD

**Discussion by vaibhavsharma-saaslabs** [saadeghi/daisyui#2280] [Q&A] [answered]
In tooltip-bottom, the tooltip is at centre of the component which also aligns my text to center.

I want tooltip at bottom left instead of bottom centre which is by default in tooltip-bottom

**Comment by stabldev** [comment, marked as answer]
hey @CodingVaibhav, you can achieve it with pseudo class:
```html
<div class="tooltip tooltip-bottom before:left-0" data-tip="hey">
```
